diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index addff26d..72d4b2f8 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -90,7 +90,7 @@ foreach(tree_type BUILD INSTALL)
     "${build_location}/roaring-config-version.cmake"
     VERSION ${ROARING_LIB_VERSION}
     COMPATIBILITY SameMajorVersion)
-  configure_package_config_file("${CMAKE_SOURCE_DIR}/cmake/config.cmake.in"
+  configure_package_config_file("${PROJECT_SOURCE_DIR}/cmake/config.cmake.in"
                                 "${build_location}/roaring-config.cmake"
                                 INSTALL_DESTINATION "${install_location}")
 
diff --git a/tools/cmake/FindOptions.cmake b/tools/cmake/FindOptions.cmake
index 69b40ed8..55cbece4 100644
--- a/tools/cmake/FindOptions.cmake
+++ b/tools/cmake/FindOptions.cmake
@@ -38,10 +38,11 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXXSTD_FLAGS} ${OPT_FLAGS} ${INCLUDE_F
 
 if(MSVC)
 add_definitions( "/W3 /D_CRT_SECURE_NO_WARNINGS /wd4005 /wd4996 /wd4267 /wd4244  /wd4113 /nologo")
-endif()
 if(MSVC_VERSION GREATER 1910)
   add_definitions("/permissive-")
 endif()
+endif()
+
 if(ROARING_LINK_STATIC)
   if(NOT MSVC)
     set(CMAKE_EXE_LINKER_FLAGS "-static")
